.github/workflows/test.yml

```diff
@@ -39,9 +40,18 @@ jobs:
       shell: bash
     - name: Install audit
       run: cargo install cargo-audit
+      shell: bash 
+    - name: Prepare to store metadata
+      run: mkdir -p src/well-known
       shell: bash
-    - name: fmt
-      run: cargo fmt
+    - name: Store JWKS metadata
+      run: echo "${{ secrets.JWKS_JSON }}" > src/well-known/jwks.json
```

Comment:
I'm presuming two things...

1. Echoing the secret to a file like this is safe to do.
2. The resulting file is being used by the new `cargo check` step.